### PR TITLE
Add Heat Index to JSON payload

### PR DESCRIPTION
**Pull request by cribbstechnologies**
I added "Real Feel" to the sensor using NOAA calculations

### CODE REVIEW

**Comment by sunnythaper** [comment]
This is awesome! Heat Index is def a feature I've wanted!

File: example_home_assistant_configuration.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-mqtt:  
+mqtpt:
```
Comment (sunnythaper) [inline example_home_assistant_configuration.yaml]: Noticed this typo. Should read "mqtt"

Comment (cribbstechnologies) [inline example_home_assistant_configuration.yaml]: damn vim

**Comment by rbflurry** [comment]
This is cool. Thank you.

**Comment by bruhautomation** [comment]
Thanks much for this code. Looks awesome!

**Pull request merged**